drivers/spi/spi_context.h

```diff
@@ -112,13 +112,27 @@ static inline void spi_context_release(struct spi_context *ctx, int status)
 static inline int spi_context_wait_for_completion(struct spi_context *ctx)
 {
 	int status = 0;
+	uint64_t ms;
```

Comment:
As @gmarull has mentioned `ms` is not a very good name. What about `timeout_ms`?